src/app/EventManagement.cpp

```diff
@@ -97,6 +97,7 @@ struct EventEnvelopeContext
     EndpointId mEndpointId   = 0;
     EventId mEventId         = 0;
     EventNumber mEventNumber = 0;
+    DataModel::Nullable<chip::FabricIndex> mFabricIndex;
```

Comment:
Again, what does it mean for this to be null?

Reply:
Updated doc in EventLogging.h.
For nullable fabric index, it means this event can be retrieved by read handler without the fabric restriction. 0 is still meaningful fabric.

Reply:
How is 0 a "meaningful fabric"?

Reply:
updated. use 0 as undefined spec. remove nullable